spec/double_entry/reporting_spec.rb

```diff
@@ -42,4 +42,146 @@
       end
     end
   end
+
+  describe '::aggregate' do
```

Comment:
Seems to be a ruby thing. `::` indicate class methods. `#` indicate instance methods.
eg in the method index in Ruby Doc http://ruby-doc.org/core-2.2.2/Array.html